receipt_dynamo/analyze_mixin_consolidation.py

```
@@ -0,0 +1,178 @@
+    print("-" * 40)
+    for count, files in sorted(mixin_counts.items()):
+        print(f"{count} mixins: {files} files")
+
+    # Calculate reduction
+    total_current = sum(count * files for count, files in mixin_counts.items())
+    total_after = (
+        len(migrations) * 1
+    )  # Each file will have 1 consolidated mixin
+    reduction = ((total_current - total_after) / total_current) * 100
+
+    print(f"\n\nInheritance reduction:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)